src/Extensions/Logging/Internal/OmexLogsEventSource.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	[EventSource(Name = "Microsoft-OMEX-Logs")]
+	internal sealed class OmexLogsEventSource : EventSource
+	{
+		static OmexLogsEventSource()
+		{
+			Process process = Process.GetCurrentProcess();
+			ProcessName = string.Format("{0} (0x{1:X4})", process.ProcessName, process.Id);
```

Comment:
>Format [](start = 24, length = 6)

`string.Format` should always take `CultureInfo` as the first parameter. This is for logs I believe, so `CultureInfo.InvariantCulture` seems like the appropriate one here to ensure the result in the same for all machines

Reply:
Fixed, thank you